### PR TITLE
New Article: Cloud Backup - Basic troubleshooting on Windows 2008 ser…

### PR DESCRIPTION
**Pull request by BadCrumpany**
…vers

Basic troubleshooting steps for Cloud Backup to assist non-technical customers.

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-bf7dbbac55/how-to/)

**Comment by cyrichardson** [comment]
@BadCrumpany I made edits in your patch to make the language in this article similar to that in your other article "Cloud Backup - Basic troubleshooting on Windows Server 2012". Please review and provide your feedback. Thanks!

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-0f450fabf9/how-to/). :bow:

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-0ee4eb6ac6/how-to/)

File: content/cloud-backup/cloud-backup-basic-troubleshooting-on-window-2008-servers.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+9. Click the drop-down menu next to **Startup type** and select (if not already selected) **Automatic**.
+
+10. Click **OK** at the bottom of the page.
+
+11. After the service is restarted, check the Control Panel again to check the backup status.
+   
+    - If the status is **connected**, try to run the backup again, and if it fails, contact
+      Rackspace Support.
+    - If the status is **disconnected**, download the backup log from within Windows on the server.
+      The log is located at C:\ProgramData\Driveclient\log\driveclient.log. To expediate the issue
```
Comment (catlook) [inline content/cloud-backup/cloud-backup-basic-troubleshooting-on-window-2008-servers.md]: do you mean "expedite"?

**Comment by BadCrumpany** [comment]
Yep, whoops!  Sorry about that.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-62d89d4a21/how-to/)

**Pull request merged**